src/views/Scenario/Scenario.js

```diff
@@ -28,8 +28,26 @@ import { AppInsights } from '../../services/AppInsights';
 import { PERMISSIONS } from '../../services/config/Permissions';
 import { PermissionsGate } from '../../components/PermissionsGate';
 import { getCreateScenarioDialogLabels, getReportLabels } from './labels';
-import useStyles from './style';
+// import useStyles from './style';
```

Comment:
Comment to remove :slightly_smiling_face:

Reply:
Thanks